src/wallet-utility.cpp

```diff
@@ -141,7 +141,7 @@ bool WalletUtilityDB::DecryptKey(const std::vector<unsigned char>& vchCryptedSec
 /*
  * Encrypted private key in WIF format
  */
-std::string WalletUtilityDB::getCryptedKey(CDataStream ssKey, CDataStream ssValue, std::string masterPass)
+std::string WalletUtilityDB::getCryptedKey(CDataStream ssKey, CDataStream ssValue, const std::string& masterPass)
 {
     mPass = masterPass.c_str();
```

Comment:
Not actually introduced by your PR, but the previous version made me worry for the validity of the `char*` for a second. I believe this looks better:
```suggestion
    mPass = masterPass;
```

Reply:
Removed cause variable is not used.

Reply:
Mh, from a quick looks it seems to me that it is passed as a parameter `SetKeyFromPassphrase()`. Am I missing something?